composer.json

```diff
@@ -10,12 +10,22 @@
             "email": "notifiers@bugsnag.com"
         }
     ],
+    "repositories": [
```

Comment:
Why are these added? If you want to checkout a branch, just do `"bugsnag/bugsnag": "dev-base-changes",`. No need for extra repos bypassing packagist.

Reply:
Trying without specifying the repositories causes an issue, as it can't seem to find the branch.